doc/whats-new.rst

```diff
@@ -113,6 +113,9 @@ Other enhancements
   By `Keisuke Fujii <https://github.com/fujiisoup>`_.
 - Added :py:meth:`~xarray.Dataset.drop_dims` (:issue:`1949`).
   By `Kevin Squire <https://github.com/kmsquire>`_.
+- Allow ``expand_dims`` method to support inserting/broadcasting dimensions
```

Comment:
sorry one other thing -- this should be moved up to 0.12.1

Reply:
No problem. Thanks